src/main/kotlin/com/oneplanet/dto/request/ParametersRequest.kt

```diff
@@ -0,0 +1,6 @@
+package com.oneplanet.dto.request
+
+import io.micronaut.core.annotation.Introspected
+
+@Introspected
+data class ParametersRequest(val energy: Int, val energy_unit: String)
```

Comment:
One field per line would be better (see the comment in `EmissionFactorResponse`)